test/result/core_test.clj

```diff
@@ -86,6 +86,12 @@
     (is (result/payment-required? result))
     (is (result/failed? result))))
 
+(deftest result-created
+  (let [ data {:dummy "dummy"}
+        result (result/created data)]
+    (is (result/created? result))
+    (is (result/succeeded? result))))
```

Comment:
I know that isn't a explicit validation but ``result/created?`` use that symbol.So if we change the symbol on ``result/created`` or ``result/created?`` the test fails (I tested it). So @lamuria you think it's necessary?